### DISCUSSION TITLE
F90 Serial number Fix?

### DISCUSSION THREAD

**Discussion by mickatcol** [jblance/mpp-solar#258] [General]
Hey, I hope someone can help me. 

I have multiple Max-8000 MPPT Solar inverters that have developed an F90 error. It seems this error has something to do with a serial number mismatch that happens after 60 days of running. I have spoken to my supplier over several weeks now but they are not being very helpful. They advised I need some communication board and software to fix this error but so far they haven't got back to me with any more details about this board. I managed to get the software from them but it doesn't seem to be working, they say I need the comm board for it to work. 

I was reading that someone used mpp-solar to reset the serial and fix the F90 error without the need for this comm board but can't find anything more on the subject. I currently use mpp-solar to log data through the RJ45 serial port on the LCD screen, can I use the same setup to issue a command to reset the serial number with mpp-solar? 

The software is Axpert Setting Serial Number Tool V1.00.
It finds my Product ID but clicking apply gives a Setting failed! I am connected via a USB to RS232 adaptor
The inverter is MPPT MAX-8000  U1 - 46.04 U2 - 12.13

Thanks for taking the time to read and any help would be much appreciated.

**Comment by Saentist** [comment]
See this one tool [CLICK](http://ftps.voltronic.com.tw/E/Easunpower-93CB312922AC474787C6/AxpertSetSN(92932205101476).7z) and [Click](http://ftps.voltronic.com.tw/E/Easunpower-93CB312922AC474787C6/AxpertSetSNWithRemote(92932206101898).7z)
Set correct number from barcode of inverter.

Some inverters do not support SN change, firmware limitation.

**Reply by Devil123323** [reply]
link is problom
